packages/server/graphql/mutations/helpers/notifications/MSTeamsNotifier.ts

```diff
@@ -0,0 +1,419 @@
+import * as AdaptiveCards from 'adaptivecards'
+import makeAppURL from 'parabol-client/utils/makeAppURL'
+import findStageById from 'parabol-client/utils/meetings/findStageById'
+import {phaseLabelLookup} from 'parabol-client/utils/meetings/lookups'
+import appOrigin from '../../../../appOrigin'
+import Meeting from '../../../../database/types/Meeting'
+import {MSTeamsNotificationEventEnum as EventEnum} from '../../../../database/types/MSTeamsNotifications'
```

Comment:
-1 We should just use the `SclackNotificationEventEnum` here as Mattermost does it.